config/manager/kustomization.yaml

```diff
@@ -16,4 +16,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: hashicorp/vault-secrets-operator
-  newTag: 0.1.0-beta
+  newTag: 0.0.0-dev
```

Comment:
Yeah, I think we need to see why we keep getting these set. I think it happens when we run kustomize.

Reply:
Going to bump the version in `Makefile`, since that seems to be the source of the recurring reversion.

Reply:
I think it's probably because of the kustomize set image call from `integration-test`, so running the tests locally keeps modifying it:

https://github.com/hashicorp/vault-secrets-operator/blob/9fe1377055e8c2ea58bb1a29bd0b4eb794e65d74/Makefile#L250-L255

We may want to move that `set image` command into the build.yaml workflow where the integration tests are run?

Reply:
> We may want to move that set image command into the build.yaml workflow where the integration tests are run?

Eh on second thought, that's probably not what we want either. It would use the wrong image in the tests.